src/components/Strawpoll.tsx

```diff
@@ -319,41 +322,53 @@ export default function Strawpoll(props: StrawpollProps) {
     switch (stage) {
       case StrawpollStage.Preparing:
         return (
-          <Button.Group fluid>
-            <Dropdown
-              basic 
-              button
-              className="purple centered"
-              upward={false}
-              options={[{
-                key: StrawpollType.Checkbox,
-                value: StrawpollType.Checkbox,
-                text: "Choose many",
-                icon: "check square"
-              }, {
-                key: StrawpollType.Radio,
-                value: StrawpollType.Radio,
-                text: "Choose one",
-                icon: "radio"
-              }]}
-              onChange={togglePollType}
-              value={type}
-            />
-            <DeleteStrawpollModal 
-              open={modalOpen} 
-              onChangeOpenState={setOpen} 
-              onConfirm={deleteStrawpoll} 
-              trigger={
-                <Button
-                  color="red"
+          <List>
+            <List.Item>
+              <Button.Group fluid>
+                <Dropdown
                   basic
-                  onClick={()=> setOpen(true)}
-                >
-                  <Icon name="delete" />Delete strawpoll?
-                </Button>
-              } 
-            />
-          </Button.Group>
+                  button
+                  className="purple centered"
+                  upward={false}
+                  options={[{
+                    key: StrawpollType.Checkbox,
+                    value: StrawpollType.Checkbox,
+                    text: "Choose many",
+                    icon: "check square"
+                  }, {
+                    key: StrawpollType.Radio,
+                    value: StrawpollType.Radio,
+                    text: "Choose one",
+                    icon: "radio"
+                  }]}
+                  onChange={togglePollType}
+                  value={type}
+                />
+                <DeleteStrawpollModal
+                  open={modalOpen}
+                  onChangeOpenState={setOpen}
+                  onConfirm={deleteStrawpoll}
+                  trigger={
+                    <Button
+                      color="red"
+                      basic
+                      onClick={()=> setOpen(true)}
+                    >
+                      <Icon name="delete" />Delete strawpoll?
+                    </Button>
+                  }
+                />
+              </Button.Group>
+            </List.Item>
+            <List.Item >
+              <Checkbox
+                label="Delegates can add options"
+                toggle
+                checked={strawpoll ? (strawpoll.addingOptionsIsPublic || false) : false}
```

Comment:
I think this could be simplified to `checked={strawpoll?.addingOptionsIsPublic}`